src-cpp/app/modules/main/view.cpp

```diff
@@ -3,24 +3,19 @@
 
 namespace Modules::Main
 {
-View::View(QObject* parent)
-    : QObject(parent)
-{
-    m_sectionModelPtr = new Shared::Models::SectionModel(this);
-}
-
 void View::load()
 {
     // Add Wallet Section to Sections model
-    auto walletSectionItem = new Shared::Models::SectionItem(WALLET_SECTION_ID,
-                                                             Shared::Models::SectionType::Wallet,
-                                                             WALLET_SECTION_NAME,
-                                                             "",
-                                                             "",
-                                                             WALLET_SECTION_ICON,
-                                                             "",
-                                                             false,
-                                                             this);
+    auto walletSectionItem =
+        new Shared::Models::SectionItem(WALLET_SECTION_ID,
+                                        Shared::Models::SectionType::Wallet,
+                                        WALLET_SECTION_NAME,
+                                        "",
+                                        "",
+                                        WALLET_SECTION_ICON,
+                                        "",
+                                        false,
+                                        this); // FIXME: that's wrong, this->bool conversion, it will not be a parent
```

Comment:
That's actually a bug, kind of urgent.

Reply:
Damnn how did I do this, will fix it in my current PR

Reply:
and most of parameters can be default